app/models/listing.rb

```diff
@@ -0,0 +1,24 @@
+class Listing < ActiveRecord::Base
+  validates :title, :description, :category_id, :max_guests,
+            :nightly_rate, :address1, :city, :state, :zip,
+            :user_id, presence: true
+  validates :shared_bathroom, :inclusion => {:in => [true, false]}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.